### PR TITLE
(DOCSP-15846): Add POJO Codec Provider to Android MongoDB Data Access Page

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-15846

### Staged Changes (Requires MongoDB Corp SSO)

- [MongoDB Data Access - all examples, and note in "Instantiate a MongoDB Collection Handle"](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-15846/sdk/android/examples/mongodb-remote-access/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/includes/steps-crud-snippets-android.yaml [hunk 1]
```diff
@@ -82,4 +82,15 @@ content: |
...
-           :language: java
\ No newline at end of file
+           :language: java
+
+  .. note:: Using Custom Classes with MongoDB
+
+     To use classes other than the built-in ``Document`` class with
+     MongoDB, you can add *codecs* to your ``MongoCollection`` instances.
+     In the above example, we add the ``PojoCodecProvider`` to support
+     Plain Old Java Objects, known as POJOs. We first include the
```
Comment (MongoCaleb) [inline source/includes/steps-crud-snippets-android.yaml:92]: consider "Plain Old Java Objects (POJOs)."

File: source/includes/steps-crud-snippets-android.yaml [hunk 2]
```diff
@@ -82,4 +82,15 @@ content: |
...
+     default codec provider by accessing the provider that the SDK
```
Comment (MongoCaleb) [inline source/includes/steps-crud-snippets-android.yaml:93]: I don't understand this sentence ("We first include..."). I've read it twice and looked at the code and I'm not quiet sure what you're saying. Is there another approach you can take?

Comment (nathan-contino-mongo) [reply source/includes/steps-crud-snippets-android.yaml]: Reworded, take another look when you get the chance.

**Review by dacharyc — APPROVED** [review]
I did not run the code, but I did read it all and no issues jumped out at me. Assuming it works as expected, everything LGTM!

File: source/includes/steps-crud-snippets-android.yaml [hunk 3]
```diff
@@ -82,4 +82,25 @@ content: |
...
+
+     To use classes other than the built-in ``Document`` class with
+     MongoDB, you can add *codecs* to your ``MongoCollection`` instances.
+     In the above example, we add the ``PojoCodecProvider`` to support
+     Plain Old Java Objects (POJOs). Custom object support requires two
+     *codec providers*:
+
+     - the default codec provider, which provides support for built-in
+       Java types (accessed through calling ``getCodecRegistry()`` on
+       a ``MongoCollection`` with no custom codecs)
```
Comment (MongoCaleb) [inline source/includes/steps-crud-snippets-android.yaml:97]: wee nit: either a period here or no period at the end of the next bullet.

**Review by MongoCaleb — APPROVED** [review]
Great change!

**Pull request merged**